marketplace/src/app/pages/home-page/home-page.component.ts

```diff
@@ -1,12 +1,33 @@
-import { Component } from '@angular/core';
+import {Component, inject} from '@angular/core';
+import {MatCardModule} from "@angular/material/card";
+import {NgForOf, NgOptimizedImage} from "@angular/common";
+import {MatDividerModule} from '@angular/material/divider';
+import {ApiDataService} from "@common/services/api-data.service";
+import {RouterLink} from "@angular/router";
 
 @Component({
   selector: 'app-home-page',
   standalone: true,
-  imports: [],
+  imports: [
+    MatCardModule,
+    NgForOf,
+    MatDividerModule,
+    NgOptimizedImage,
+    RouterLink
+  ],
   templateUrl: './home-page.component.html',
   styleUrl: './home-page.component.scss'
 })
 export class HomePageComponent {
+  private apiService = inject(ApiDataService);
 
+  apiList: ApiDetails[] = [];
+
+  apiData = this.apiService.getApiInformationData().subscribe(apis => this.apiList = apis as ApiDetails[])
```

Comment:
Das könntest du zu nem observable machen also:
apiData$: Observable<APIDetails[]> = this.apiService.getApiInformationData();

und dann im template mit | async subscriben dann brauchst du dich nicht ums unsubscribe kümmern. Aber ist ja hier noch nicht so relevant